package.json

```diff
@@ -17,6 +17,13 @@
     "type": "git",
     "url": "git+https://github.com/gravity-ui/components.git"
   },
+  "exports": {
+    "./unstable_filters": {
```

Comment:
may be use as stable in a normal way?
We have already tested this api and made fixed after previous PR

Reply:
I'd like to publish it first and change to stable version in nearest release